pkg/util/iptables/fake_iptables.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package iptables
```

Comment:
I must be missing something - this looks exactly the same as the last time I looked at it.

Here's what I am proposing:

mkdir pkg/util/iptables/testing
git mv pkg/util/iptables/fake_iptables.go pkg/util/iptables/testing/fake.go
rename the package line in fake.go
import pkg/util/iptables/testing in hollow* and call ipttest.NewFake()

Is this not viable for some reason I am not seeing?